docs/Updating-npm-bundled-node-gyp.md

```diff
@@ -1,63 +1,40 @@
-`npm` bundles its own, internal, copy of `node-gyp`. This internal copy is independent of any globally installed copy of node-gyp that
-you may have installed via `npm install -g node-gyp`.
+npm ships with its own version of node-gyp. Older versions of npm have olderversions of node-gyp. npm has caught up recently to node-gyp's release cycle butit's expected we'll get out of sync and there will be cases where you need an oldnpm but a newer node-gyp, or npm doesn't ship yet with a node-gyp that we'vereleased and it has fixes you need.
```

Comment:
My sense is that this sentence is more meandering and less clear than the original.

Reply:
agree